server/entity/lightning.go

```diff
@@ -70,8 +79,38 @@ func (li *Lightning) New(pos mgl64.Vec3) world.Entity {
 	return NewLightning(pos)
 }
 
+// WithoutDamage returns the lightning entity with damage disabled, it won't damage any entities.
+func (li *Lightning) WithoutDamage() *Lightning {
+	li.mu.Lock()
+	li.damage = false
+	li.mu.Unlock()
+	return li
+}
+
+// WithoutBlockFire returns the lightning entity but won't set any blocks on fire.
+func (li *Lightning) WithoutBlockFire() *Lightning {
```

Comment:
I think it might be better to add additional `New*` methods for these, like `NewLightningWithDamage` and maybe a bool in that to disable fire.

Reply:
The fire bool would have to be passed to the default NewLightning too, so it would make more sense to just pass everything to NewLightning, but that looks weird so from what i see this is the best solution

Reply:
Just choose a default for `NewLightning` and only accept a fire bool in `NewLightningWithDamage`.